package.json

```diff
@@ -29,7 +29,8 @@
     "jstransform": "^11.0.0"
   },
   "devDependencies": {
-    "babel-core": "^5.2.7",
+    "babel": "^5.3.1",
+    "babel-core": "^5.3.1",
```

Comment:
So these don't get out of sync, want to just replace our uses of `require('babel-core')` with `require('babel')`? Their node API is identical.